### PR TITLE
Add skew and trading history

### PR DESCRIPTION
**Pull request by LeifuChen**
## Description
Implement a trade history & skew panel

## Related issue
#639 

## Motivation and Context
Show the trade history and skew panel.

## How Has This Been Tested?
1. Switch to the different markets and check the skew and trade history
3. Check the layout when there is no trade

## Screenshots (if appropriate):
image
image
image

### CODE REVIEW

File: queries/futures/types.ts [hunk 1]
```diff
@@ -163,9 +163,11 @@ export type FuturesTradeWithPrice = {
 };
 
 export type FuturesTrade = {
-	size: string;
+	size: Wei;
 	asset: string;
-	price?: string;
+	price?: Wei;
+	trxId: string;
```
Comment (Tburm) [inline queries/futures/types.ts:169]: Is `trxId` correct? Other places we use `txnHash`

File: queries/futures/useGetFuturesTrades.ts [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+						asset: true,
+					}
+				);
+				return response ? mapTrades(response) : null;
+			} catch (e) {
+				console.log(e);
+				return null;
+			}
+		},
+		{ enabled: isAppReady && isL2 && !!walletAddress, ...options }
```
Comment (Tburm) [inline queries/futures/useGetFuturesTrades.ts:54]: `!!walletAddress` wll prevent this running before they are connected

File: queries/futures/useGetFuturesTrades.ts [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+import { getFuturesTrades } from './subgraph';
+import { DEFAULT_NUMBER_OF_TRADES } from 'constants/defaults';
+
+const useGetFuturesTrades = (
+	currencyKey: string | undefined,
+	options?: UseQueryOptions<FuturesTrade[] | null>
+) => {
+	const isAppReady = useRecoilValue(appReadyState);
+	const isL2 = useRecoilValue(isL2State);
+	const walletAddress = useRecoilValue(walletAddressState);
```
Comment (Tburm) [inline queries/futures/useGetFuturesTrades.ts:20]: can remove this

**Review by Tburm — COMMENTED** [review]
Minor changes requested, but we should definitely populate this if a wallet isn't connected.

One other comment that I think Yashar had as well, the color of the scroll bar is kind of obnoxious. Can we make the background of the scrollbar transparent or something? :
image

File: queries/futures/useGetFuturesTrades.ts (hunk 2 above)
Comment (LeifuChen) [reply queries/futures/useGetFuturesTrades.ts:54]: Yes. Thanks for your catch. I fixed this. There is no need to connect the wallet to show the trades history.

File: queries/futures/types.ts (hunk 1 above)
Comment (LeifuChen) [reply queries/futures/types.ts:169]: well, it's `id` in query so I just use `trxId`. Will change it to `trxHash`.

File: queries/futures/useGetFuturesTrades.ts (hunk 3 above)
Comment (LeifuChen) [reply queries/futures/useGetFuturesTrades.ts:20]: Yes. Thanks.

**Comment by LeifuChen** [comment]
@Tburm @platschi  The following changes have been made.
- [X] show data even the wallet is disconnected 
- [X] remove the placeholder row when no trades 
- [X] remove the extra padding space of the table row 
- [X] fix the shifted layout issue
- [X] rename the query type

**Review by Tburm — APPROVED** [review]

**Pull request merged**